tests/dataset_histograms/computing_histograms_test.py

```diff
@@ -563,28 +570,164 @@ def test_compute_partitions_privacy_id_count_histogram(
                          histogram.name)
         self.assertListEqual(expected, histogram.bins)
 
+    @parameterized.product(
+        (
+            dict(testcase_name='empty', input=lambda: [], expected=lambda: []),
+            dict(
+                testcase_name='small_histogram',
+                input=lambda: [((1, 1), 0.5), ((1, 2), 1.5), (
+                    (2, 1), -2.5), (
+                        (1, 1), 0.5)],  # ((privacy_id, partition), value)
+                expected=lambda: [
+                    hist.FrequencyBin(
+                        lower=-1.5, upper=-1.4997, count=1, sum=-1.5, max=-1.5),
```

Comment:
nit: can be add a comment what the step in bucketing is to be able to verify that lowers and uppers are correct

Reply:
ah, I see, the step is below, maybe move it higher, but idk, current code is ok as well